templates/e-commerce/partials/blog-preview.html

```diff
@@ -0,0 +1,11 @@
+{{#articles}}
+<article class="md-pb7">
+  <amp-img class="mb3 md-col-10" src="{{images.0.narrow}}" srcset="{{images.0.narrow}} 750w,{{images.0.wide}} 1280w" width="{{imageWidth}}" height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
```

Comment:
if narrow and wide are set on all elements in the array, you should not have to worry about this right? 
It is not great to rely on something like this here because of some one replaces your first image they might not expect this consequence. 

```json
{
  "articles": [
    {
       "image": {
         "narrow": "",
         "wide": "",
         "title": ""
        }
    }
  ]
}
```